### DISCUSSION TITLE
Is it posible to Grant IAM Permission to a user?

### DISCUSSION THREAD

**Discussion by javijvj** [cyrilgdn/terraform-provider-postgresql#167] [Q&A]
Hello, 

I would like to run the following command, but I don't know how would be the right way to coding it using this provider, because the `postgresql_grant` resource requires multiples args which are not defined in the command that I need to rum.

Command: 
```
GRANT rds_iam TO mydbuser;
```

I would appreciate your suggestions.

**Comment by nitrocode** [comment]
It looks like this is available like this

```hcl
resource "postgresql_role" "test_role" {
  name  = "test_role"
  roles = ["rds_iam"]
}
```

References

* https://www.reddit.com/r/Terraform/comments/hxvwm8/grant_a_role_to_a_user_in_postgres/
* https://github.com/hashicorp/terraform-provider-postgresql/pull/52
* https://registry.terraform.io/providers/cyrilgdn/postgresql/latest/docs/resources/postgresql_role
* https://github.com/hashicorp/terraform-provider-postgresql/issues/21
